### ISSUE TITLE
Syntax Error

### ISSUE THREAD

**Issue by KrisCarr** [bschwarz/puml-themes#3]
Importing the .puml file for the following tested themes results in a "Syntax Error?" 

I assume the same error happens on all themes.

Environment:
OS: Windows 10 x64
PlantUML: 1.2020.8
Graphvix: 2.38 Stable Release
Java: 14.0.1 2020-04-14 x64

Themes tried:
### Cyborg-Outline
!include https://raw.githubusercontent.com/bschwarz/puml-themes/master/themes/cyborg-outline/puml-theme-cyborg-outline.puml

```failure on line 70 !procedure $success($msg)```

### Cerulean
!include https://raw.githubusercontent.com/bschwarz/puml-themes/master/themes/cerulean/puml-theme-cerulean.puml

```failure on line 66 !procedure $success($msg)```

### Hacker

!include https://raw.githubusercontent.com/bschwarz/puml-themes/master/themes/hacker/puml-theme-hacker.puml

```failure on line 40 !procedure $success($msg)```

**Comment by bschwarz** [comment]
Hi Kris,

Thanks for the feedback. However, I can not reproduce the error. Can you verify you are using a recent plantuml jar file? I tried both on Windows and Linux, and it works. You can also see that it works on the online plantuml server:

http://www.plantuml.com/plantuml/uml/HOjDhi8m30RtEKNV0LBh0IjMxL7IrDGL-ObiHv4uFOb0OZSp-hv5rB7M6CpVcNoeEqwaMkGEmAxDnwbKjoh8FYV5fBFF4JRnr1o_eFIZLSA80j698iFNoFbdjz_0VieyYZ7hP6knq_e_YC-3jCjbo9vRwdmT7F0n-iqicFR-UmC0

Cheers,
Brett

**Comment by vlasebian** [comment]
Hello!

I have encountered the same problem. I am using plantuml.jar to generate diagrams. This is the version output:
```
PlantUML version 1.2020.06 (Sun Apr 05 15:38:32 EEST 2020)
(GPL source distribution)
Java Runtime: OpenJDK Runtime Environment
JVM: OpenJDK 64-Bit Server VM
Java Version: 1.8.0_252-8u252-b09-1~19.10-b09
Operating System: Linux
Default Encoding: UTF-8
Language: en
Country: US
Machine: explorer
PLANTUML_LIMIT_SIZE: 4096
Processors: 4
Max Memory: 3,685,744,640
Total Memory: 249,561,088
Free Memory: 243,997,440
Used Memory: 5,563,648
Thread Active Count: 1

The environment variable GRAPHVIZ_DOT has not been set
Dot executable is /usr/bin/dot
Dot version: dot - graphviz version 2.40.1 (20161225.0304)
Installation seems OK. File generation OK
```
Here is the source for the diagram that gives me the error when trying to include Cerulean theme:
```
@startuml

!include https://raw.githubusercontent.com/bschwarz/puml-themes/master/themes/cerulean/puml-theme-cerulean.puml

participant Client
participant Server
title Message exchange
skinparam wrapWidth 200
skinparam sequenceMessageAlign center
autonumber 1 "<b>0." 
scale 2

|||
Client -> Server: WriteRequest on 0x003d
note over Client, Server
Value: 00 00
end note

|||
Server --> Client: WriteResponse from 0x003d
note over Client, Server
end note

@enduml
```
and here is the error:
```
!procedure $success($msg)
Syntax Error?
```

Any suggestions on this? I'd really like using this theme, it looks great!

**Comment by bschwarz** [comment]
Hi,

This is a version thing. procedures are relatively new. I tried at one point to test for different versions in the the config file, but I couldn't get it to work. I will try again, but probably not until next week. It's hard too because some versions were buggy. I was also thinking about having multiple versions of puml-themes, to support older jar files, but that will take some time to build out.

Are you able to upgrade the plantuml.jar file? I think the latest is 1.2020.08. I just tested with that version and your puml works. You can also test it out here: http://www.plantuml.com/plantuml/uml/XO-zJiCm58LtFyMbCqbCsIfK5J6pW4Hdnpaa5lPriAzfXVhmf3I5CY5vi8--y-DD5fEa1A_KbMFhImSQHCQyQffaTdNlP2XjoKWsie2bjZ4qRRR3pgJFPfoSbGm8o4qmMP2Q-MMHYeVX2wGwQ_NuNJYQ9CwwqR3GWtTW-ICz8tqWAN7YGO_8sVGWxErWk8VARuudsWJQ9JDkNIS3BRM-q3FU2zXYjjvxrpDPi20fKoHo2IqItT3YhbthUa4gM-D1IwKEXuCwBQ9gFGzPqJOvmTCnDGj59htN-hPJ70KK9sB-nFLv-Oln1IlIUZeAtD6HF8MV2AggzMpwISzZv0nwJJ7yM_2RkPck9VWl

If you can't upgrade jar files, then an interim solution would be to download the cerulean puml file, and change all of the instances of  _!procedure_ to _!function_ and _!endprocedure_ to _!endfunction_. Then you can either add _!include puml-theme-cerulean.puml_ to your puml file, or run the plantuml like this:

java -jar plantuml.jar -config <path to file>/puml-theme-cerulean.puml

**Comment by vlasebian** [comment]
I upgraded the plantuml.jar file and now it works. Thank you!

**Comment by KrisCarr** [comment]
Thanks guys. What seems to have been happening in my case is the Visual Studio Code plugin for PlantUML bundling its own version out-the-box and being awkward about switching to a non-bundled version, sometimes using internal, sometimes using the one I define in the settings for previewing. I have it working now in a command-line outside the IDE which is good enough for the time being until I figure out why VS Code is being awkward.

**Comment by bschwarz** [comment]
ok, I am glad you guys figured something out. I am going to close this issue, but feel free to open another issue if you still have problems.
